src/implementations/twig/components/ecl-component-button/__snapshots__/button.test.js.snap

```diff
@@ -154,7 +154,7 @@ exports[`EC - Button Primary renders correctly 1`] = `
 exports[`EC - Button Search renders correctly 1`] = `
 <button
   class="ecl-button ecl-button--search"
-  type="submit"
+  type="button"
```

Comment:
Are you sure that this change is desired?

Reply:
Yep, i've checked them one by one, i will then check also in ecl-twig the reason why of the "wrong" values for the type in the test, but here they are correct.

Reply:
well, in the case of a search form for instance, the button should remain a "submit" I think

Reply:
But do we have to discuss here this? I mean, in ECL, in production, the search button example is a type="button"
![image](https://user-images.githubusercontent.com/14326885/95302523-5a60dc00-0882-11eb-9473-e79a61ac8004.png)

I think it would be in case more relevant for V2 than V3, then, here we are for now just using these templates to play with the structure, i believe, not defining the new V3 version of them.
 
Whatever, for me, if you want to have it as as submit here, it's a line to be changed in the relative specs..